sdk-base/api/metalava.txt

```diff
@@ -1,4 +1,11 @@
 // Signature format: 3.0
+package com.mapbox.common {
+
+  public final class MapboxCommonLoggerKt {
+  }
+
+}
+
```

Comment:
can we get rid of this API surface? like move the internal functions to a internal object instead of a plain kotlin file?